src/lib/contents/coder.ts

```diff
@@ -0,0 +1,352 @@
+        "<a href='/screencasts/continuously-prebuild-your-project'>https://www.gitpod.io/screencasts/continuously-prebuild-your-project</a>",
+    },
+  },
+  {
+    type: "tooltip",
+    data: {
+      text: "Snapshots",
+      tooltip:
+        "<a href='https://www.gitpod.io/screencasts/collaborating-with-your-team'>https://www.gitpod.io/screencasts/collaborating-with-your-team</a>",
+    },
+  },
+];
+
+const gitpodColum: FeatureTableColumn = {
+  isHighlighted: true,
+  header: {
+    headline: "Gitpod",
+    image: {
+      alt: "gitpod",
+      path: "/images/gitpod-logo.svg",
+    },
+  },
+  enteries: [
+    {
+      items: [
+        {
+          term: "License",
+          text: "Open-source",
+        },
+        {
+          term: "Architecture",
+          text: "Secure Containers",
+        },
+        {
+          term: "Made for teams",
+          availability: true,
+        },
+        {
+          term: "Made for individuals",
+          availability: true,
+        },
+        {
+          term: "Pricing Saas",
+          text: "Free for 50h/month",
+        },
+        {
+          term: "Pircing Self-hosted",
+          text: "Free tier available",
+        },
+        {
+          term: "GitLab Integration",
+          availability: true,
+        },
+        {
+          term: "GitHub Integration",
+          availability: true,
+        },
+        {
+          term: "Bitbucket Integration",
+          availability: true,
+        },
+        {
+          term: "Self-host on GCP",
+          availability: true,
+        },
+        {
+          term: "Self-host on AWS",
+          availability: true,
+        },
+        {
+          term: "Self-host on Azure",
+          availability: true,
+        },
+        {
+          term: "Self-host on Azure",
+          availability: true,
+        },
+        {
+          term: "Self-host on Kubernetes",
+          availability: true,
+        },
+        {
+          term: "Reproducible environments",
+          availability: true,
+        },
+        {
+          term: "Collaboration & Sharing",
+          availability: true,
+        },
+        {
+          term: "Fast developer onboarding",
+          availability: true,
+        },
+        {
+          term: "Prebuilds",
+          availability: true,
+        },
+        {
+          term: "Snapshots",
+          availability: true,
+        },
+      ],
+    },
+  ],
+};
+
+const coderColumn: FeatureTableColumn = {
+  header: {
+    headline: "Coder",
+    image: Coder,
+  },
+  enteries: [
+    {
+      items: [
+        {
+          term: "License",
+          text: "Open-source",
+        },
+        {
+          term: "Architecture",
+          text: "Virtual Machines",
+        },
+        {
+          term: "Made for teams",
+          availability: true,
+        },
+        {
+          term: "Made for individuals",
+          availability: false,
+        },
+        {
+          term: "Pricing Saas",
```

Comment:
I know, it won't be visible to the user. But, it should be consistent across the site.

```suggestion
          term: "Pricing (SaaS)",
```